Index: src/main/java/lt/bongibau/behelpfull/authentication/Authentication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/lt/bongibau/behelpfull/authentication/Authentication.java b/src/main/java/lt/bongibau/behelpfull/authentication/Authentication.java
new file mode 100644
--- /dev/null	(date 1728315013517)
+++ b/src/main/java/lt/bongibau/behelpfull/authentication/Authentication.java	(date 1728315013517)
@@ -0,0 +1,11 @@
+package lt.bongibau.behelpfull.authentication;
+
+import lt.bongibau.behelpfull.models.User;
+import lt.bongibau.behelpfull.users.UserManager;
+
+public class Authentication {
+
+    public User login(String username, String password) {
+        UserManager.getInstance().getUser()
+    }
+}
